packages/visualizations/src/components/Map/MapRender.svelte

```diff
@@ -151,7 +153,7 @@
         }
     }
 
-    function addTooltip(e: MapMouseEvent) {
+    function addTooltip(e: MapLayerMouseEvent) {
         // @ts-ignore // Somehow `features` isn't part of the type, but exists in the object at runtime
```

Comment:
I may have missed something, and I apologise in advance if that's the case. But couldn't we delete `@ts-ignore` and do a `if (e.features)` check instead?

Reply:
Absolutely, you're right. Even my comment was outdated from before the `MapLayerMouseEvent` change, there was still an error for a different reason, I just didn't even look at it 😬

Changed in 20727d3d1480fa6113464d7592f370e7d4f53d5a